cms/static/js/views/modals/move_xblock_modal.js

```diff
@@ -135,13 +135,20 @@ function($, Backbone, _, gettext, BaseView, XBlockViewUtils, MoveXBlockUtils, Ht
                 $moveButton.addClass('is-disabled');
             }
         },
+        validCategoryCheck: function(targetParentType, sourceParentType) {
+            if (!_.contains(['course, chapter, sequential, vertical'], sourceParentType)) {
+                sourceParentType = 'vertical';  // eslint-disable-line no-param-reassign
+            }
+            return targetParentType === sourceParentType;
+        },
```

Comment:
@mushtaqak Instead of hardcoding the array here, what about the below code?
```JS
if (!_.contains(_.keys(this.moveXBlockListView.categoryRelationMap), sourceParentType)) {
```